portfolio/src/main/webapp/bio.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div class="topnav">
```

Comment:
As we go forward in the program, keep an eye out for a way to share this navigation bit that's common across your pages. That way you don't have to maintain each copy separately :)

Reply:
I will look into. I saw something about creating a navtab.html file and using JavaScript to implement it on all pages but I have to read about it more to full understand it.